apps/api/test/setup.ts

```diff
@@ -0,0 +1,23 @@
+import "tsconfig-paths/register";
+
+import { createClient } from "redis";
+import { TestManager } from "./TestManager";
+
+export default async (): Promise<void> => {
+    console.log("# Started Jest globalSetup.");
+    const testManager = new TestManager();
+
+    await testManager.beforeAll();
+
+    await testManager.app.init();
+
+    // TODO: Apply Database migrations/seeders.
```

Comment:
Where are the db stuff?

Reply:
Hmm... I can't recall why I didn't add them, I can work on them this weekend, but I prefer that we merge this anway.

Reply:
I spent about ~1 hour trying to figure this out. Two issues created regarding this.
https://github.com/activecourses/disworse/issues/76
https://github.com/activecourses/disworse/issues/77